app.js

```diff
@@ -18,40 +19,55 @@ go();
 async function go() {
   try {
     for (const org of config.orgs) {
+      console.log('⏳ Collecting repos for', org)
       let page = 1;
       while (true) {
         const params = {
           page,
-          access_token: config.token
+          sort
```

Comment:
Added sort to sort by updated date while avoiding a break. Sorting by updated date speeds up getting what we actually need.